src/haxelib/api/Repository.hx

```diff
@@ -180,6 +181,9 @@ class Repository {
 		}
 
 		FsUtils.deleteRec(versionPath);
+		
+		var versions = getProjectInstallationInfo(name).versions.pop();
+		setCurrentVersion(name, versions);
```

Comment:
We really don't want that. Especially since you're also doing it when removing a version that is not the current version.

Reply:
ah I was incorrect with this one yes. 
The intention was for the case of using a dev version and if you were to remove the current version, it doesn't change anything in the `.current` file.

If you're on say `flixel 5.0.0` (`cat flixel/.current` -> `5.0.0`), begin using a dev directory for flixel, do something like `haxelib remove flixel 5.0.0`, we want to properly update `.current` so if you disable the dev, it leads to whatever the most recent directory is `versions.pop()`.

However you're right, my implementation is incorrect since we don't want to remove / set the current version unconditionally!

Reply:
how does something like 
```hx
FsUtils.deleteRec(versionPath);
		
if (current == version)
{
	var versions = getProjectInstallationInfo(name).versions.pop();
	setCurrentVersion(name, versions);
}
```
shake out? Unless I'm still misunderstanding 🙇‍♂️
In this snippet, the code will run if `.dev` DOES exist (otherwise we would have thrown an error earlier). So if we have a `.dev` haxelib, and we just removed our `.current` set haxelib, we update the `.current` file to the most recent version we have installed. Again, only if we are:
- uninstalling our "current" version
- we are using a `haxelib dev` based haxelib
- there are other versions installed